active/gridsyntax/GridSyntaxSpec2.md

```diff
@@ -0,0 +1,368 @@
+```
+
+## Grid with custom column and row features
+
+This example is equivalent to the previous one, except one column has a minimum width and one row has a minimum height, which requires the more complete property element syntax.
+
+```xml
+<Grid>
+    <Grid.ColumnDefinitions>
+          <ColumnDefinition Width="1*" MinWidth="100" />
+          <ColumnDefinition Width="2*" />
+          <ColumnDefinition Width="Auto" />
+          <ColumnDefinition Width="*" />
+          <ColumnDefinition Width="300" />
+    </Grid.ColumnDefinitions>
+    <Grid.RowDefinitions>
+          <RowDefinition Height="1*" MinHeight="100" />
+          <RowDefinition Height="Auto" />
+          <RowDefinition Height="25" />
+          <RowDefinition Height= "14" />
+          <RowDefinition Height="20" />
+    </Grid.RowDefinitions>
+</Grid>
+```
+
+
+# Remarks
+
+# API Details
+
+## ColumnDefinition API
+
+```cs
+[webhosthidden]
+
+// vvv
+[contentproperty("Width")] // Sets the ColumnDefinition's content property to the Width property.
+// ^^^
+
+runtimeclass ColumnDefinition : Windows.UI.Xaml.DependencyObject
+{
+    ColumnDefinition();
+    Windows.UI.Xaml.GridLength Width;
+    Double MaxWidth;
+    Double MinWidth;
+    Double ActualWidth{ get; };
+    static Windows.UI.Xaml.DependencyProperty WidthProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MaxWidthProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MinWidthProperty{ get; };
+};
+```
+
+## RowDefinition API
+
+```cs
+[webhosthidden]
+
+// vvv
+[contentproperty("Height")] // Sets the RowDefinition's content property to the Height property.
+// ^^^
+
+runtimeclass RowDefinition : Windows.UI.Xaml.DependencyObject
+{
+    RowDefinition();
+    Windows.UI.Xaml.GridLength Height;
+    Double MaxHeight;
+    Double MinHeight;
+    Double ActualHeight{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeightProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MaxHeightProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MinHeightProperty{ get; };
+};
+```
+
+# Appendix
+
+## Alternatives
+
+## Syntax Considerations and Rationale
+In developing the syntax, the team considered developer familiarty and patterns as much as possible to ensure that the Grid learning curve could be decreased. One of the considerations was how similar the syntax should be to JSON, as that's a popular notation style that often addresses the same problems that Xaml does. 
+
+JSON defines their arrays as: ```"words" : ["apple", "banana", "cherry"]```
+
+Xaml defines a collection-typed property as: ```<Words="'Apple', 'Banana', 'Cherry'"/>```
+
+The syntax that we chose encloses strings individually in quotes, as JSON does, but it does so in a way that is more consistent with existing Xaml style and Xaml functions. An example of this syntax used elsewhere in Xaml is for the x:Bind syntax:
+
+```xml <TextBlock Text="{x:Bind VM.MyText, FallbackValue='Hello, world', Mode=TwoWay}" />```
```

Comment:
Need to split onto three lines. Cramming into one line doesn't work because the `xml` is being interpreted as literal text rather than a syntax highlighter hint. It also confuses github's markdown syntax-highlighter.

~~~
```xml
<TextBlock Text="..." />
``` 
~~~